backend/src/main/routes/UserRoute.ts

```diff
@@ -22,6 +24,13 @@ export default class UserRoute extends CommonRoutesConfig {
   configureRoutes(): express.Application {
     this.getApp()
       .route('/users')
+      // Example of restriction on route
```

Comment:
We can delete this file entirely. I created it as an example but we have many files now so no need.

Reply:
We should still keep it since it shows an example of how to restrict the endpoints

Reply:
Fair points